packages/plugin-eslint/src/lib/eslint-plugin.spec.ts

```diff
@@ -1,7 +1,51 @@
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+import type { SpyInstance } from 'vitest';
 import { eslintPlugin } from './eslint-plugin';
 
 describe('eslintPlugin', () => {
```

Comment:
Scenario: There is a project which has multiple root ESLint configs, e.g. separately for production and test code. The ESLint rules should be applied on the same monorepo but different files and be potentially managed by the same team. Should multiple plugins be created or will the configuration of the plugin correctly work for such configuration?

Reply:
Yes, that can be achieved with ESLint's glob-based configs. It could be valuable to create some helper function that creates such a config for an Nx monorepo, but that's out of scope of this PR.